game-headed/src/main/java/org/triplea/game/headed/runner/HeadedGameRunner.java

```diff
@@ -12,6 +12,9 @@ private HeadedGameRunner() {}
    * Entry point for running a new headed game client.
    */
   public static void main(final String[] args) {
-    GameRunner.main(args);
+    GameRunner.start(GameRunner.Context.builder()
+        .args(args)
+        .mainClass(HeadedGameRunner.class)
+        .build());
```

Comment:
I think using a builder for 2 arguments is overkill here, simple constructor will do already.

Reply:
I have plans for `GameRunner$Context` in a few forthcoming PRs.  It's basically how I see slowly migrating code from `game-core` to `game-headed` in order to inject dependencies from `game-headed` back into `game-core`.  So, the number of fields in `Context` will probably grow relatively quickly.  I think when you see the next PR I have planned for `game-headed` migration, it will become clearer.  If you disagree at that time, I'll be happy to either get rid of the builder or even get rid of `Context` completely.

Note that once the migration is complete (will it ever be complete? :smile:), `Context` will actually disappear because there will be no further need for the DI (at least for the purposes of what it's being used for now).

Reply:
> will it ever be complete? 😄

Probably not, something completely unforseeable will cause us to revert all the changes just before the migration has been finished :P

> So, the number of fields in Context will probably grow relatively quickly. I think when you see the next PR I have planned for game-headed migration, it will become clearer.

Ah ok makes sense.
In this case I'm going to be fine with this decision in the future ^^